integration-test/background/click-to-load-youtube.js

```diff
@@ -0,0 +1,335 @@
+const fs = require('fs')
+const path = require('path')
+const harness = require('../helpers/harness')
+const { logPageRequests } = require('../helpers/requests')
+const { loadTestConfig, unloadTestConfig } = require('../helpers/testConfig')
+const { getObjectSchema } = require('../helpers/objectSchema')
+
+const testSite = 'https://privacy-test-pages.glitch.me/privacy-protections/youtube-click-to-load/'
+const testConfig = {
+    'dbg.tds.tds.trackers.youtube\\.com': {
```

Comment:
why only last `.` is escaped?
should we move that config to external file that we load here (like `youtube-iframe-api.json`)?

Reply:
This change already landed, but the last `.` is escaped because that's part of the property's name. In other words the Object we're referring to is `window['dbg']['tds']['tds']['trackers']['youtube.com']`.